### PR TITLE
Security Medical Kit

### PR DESCRIPTION
**Pull request by KillerTheFlareon**
## Short description

A simple addition of a reskinned and bigger combat medkit made for security use that can only hold medical items.
## Why we need to add this

I think it looks cool, and it helps a bit with space management as brigmedic as you get a round-start empty medkit to stuff your meds into that fits nicely in the 2x2 hole left in your belt.
## Media (Video/Screenshots)

https://github.com/user-attachments/assets/848ebe89-d474-428d-81a3-e1772ebd18e2

## Checks

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Killer Tamashi
- add: Added BrigKit to roundstart Brigmedics.

### CODE REVIEW

File: Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -179,6 +179,7 @@
         - DiscreteHealthAnalyzer
         - SurgeryTool
         - Dropper
+        - Medkit
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml]:
```suggestion

```

File: Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml [hunk 2]
```diff
@@ -191,4 +192,5 @@
         - Injector
         - Pill
         - HandLabeler
-  - type: Appearance
\ No newline at end of file
+        - SecurityMedkit
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml]:
```suggestion
```

File: Resources/Prototypes/_StarLight/Catalog/Fills/Items/belt.yml [hunk 3]
```diff
@@ -22,4 +22,5 @@
       - id: Handcuffs    
       - id: Brutepack
       - id: Ointment
-      - id: Gauze
\ No newline at end of file
+      - id: Gauze
+      - id: SecurityMedkit
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Catalog/Fills/Items/belt.yml]:
```suggestion
```

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube-inhand-left.png
Comment (Rinary1) [inline Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube-inhand-left.png:1]: Why this not in _Starlight?

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube-inhand-right.png
Comment (Rinary1) [inline Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube-inhand-right.png:1]: Why this not in _Starlight?

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png
Comment (Rinary1) [inline Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png:1]: Why this not in _Starlight?

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png
Comment (KillerTheFlareon) [reply Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png:1]: The same reason the new topicals are not in _Starlight, I'd guess? There's no place for them afaik

File: Resources/Prototypes/_StarLight/Catalog/Fills/Items/belt.yml (hunk 3 above)
Comment (KillerTheFlareon) [reply Resources/Prototypes/_StarLight/Catalog/Fills/Items/belt.yml]: You don't want it to be able to fit in the belt?

Comment (Rinary1) [reply Resources/Prototypes/_StarLight/Catalog/Fills/Items/belt.yml]: > You don't want it to be able to fit in the belt?

yes

File: Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png
Comment (Rinary1) [reply Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png:1]: > The same reason the new topicals are not in _Starlight, I'd guess? There's no place for them afaik

bruh, create it

Comment (KillerTheFlareon) [reply Resources/Textures/Objects/Specific/Medical/firstaidkits.rsi/BrigKube.png:1]: Aight.

File: Resources/Textures/_Starlight/Objects/Specific/Medical/firstaidkits.rsi/meta.json
Comment (Rinary1) [inline Resources/Textures/_Starlight/Objects/Specific/Medical/firstaidkits.rsi/meta.json:1]: license? 
copyright?

**Comment by KillerTheFlareon** [comment]
Tested. Brigmedic spawns with brigkit in backpack and the files have been moved to the folders within the _Starlight directory

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: Resources/Textures/_Starlight/Objects/Specific/Medical/firstaidkits.rsi/meta.json
Comment (KillerTheFlareon) [reply Resources/Textures/_Starlight/Objects/Specific/Medical/firstaidkits.rsi/meta.json:1]: oop. forgot, one sec.

Comment (KillerTheFlareon) [reply Resources/Textures/_Starlight/Objects/Specific/Medical/firstaidkits.rsi/meta.json:1]: fixed.

**Comment by Rinary1** [comment]
merge master(Starlight), fix crlf error

**Comment by KillerTheFlareon** [comment]
Yeah I don't know how to fix the CRLF error. Can someone look at the file?

**Pull request merged**